index.js

```diff
@@ -4,6 +4,9 @@ const fs = require('fs');
 const commonDir = require('commondir');
 const pkgDir = require('pkg-dir');
 const makeDir = require('make-dir');
+const yn = require('yn');
```

Comment:
`yn` is meant for CLI prompts. Just use the words I mentioned directly.

Reply:
Fixed!